app/scripts/services/browserStorage.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .factory('BrowserStorage', function() {
+
+    // Automaticaly namespaces all of our storage, wether in session or local
+    var namespace = 'openshift-';
+
+    function Store(type, key) {
```

Comment:
I'm not sure we want a separate `Store` object for every key. It doesn't match how we've used local/session storage to this point where we have frequently have object UIDs in the key names.

We might want to roll this back for now and keep it specific to the events service.